content/collective_intro.tex

```diff
@@ -1,11 +1,55 @@
-\emph{Collective routines} are defined as communication or synchronization
-operations on a group of \acp{PE} called an active set. The collective
-routines require all \acp{PE} in the active set to simultaneously call the
-routine.  A \ac{PE} that is not in the active set calling the collective
-routine results in undefined behavior.  All collective routines have an
+\emph{Collective routines} are defined as \newtext{coordinated} communication or
+synchronization operations performed \newtext{by} \oldtext{on} a group of \acp{PE} \oldtext{called an active set}.
+{\color{ForestGreen} % <start-newtext>
+\openshmem provides collective routines that operate on teams
+(see Section~\ref{subsec:team} on team management)
+\DeprecateInline{and active sets}.
```

Comment:
It would help set up for subsequent discussion to have a bit more text here. Something like:

OpenSHMEM provides collective routines of two types
- Collective routines that operate on teams use a team handle argument to determine which PEs will particapate in the routine, and use resources internal to the team to perform operations. These routines are to be the standard for OpenSHMEM moving forward.
- Collective routines that operation on active sets use a set of parameters to determine which PEs will participate and what resources are used to perform operations. These routines are the legacy API for collectives which will be deprecated and phased out of implementations moving forward.